src/node.cc

```diff
@@ -3589,6 +3589,10 @@ static void ParseArgs(int* argc,
   const char** new_v8_argv = new const char*[nargs];
   const char** new_argv = new const char*[nargs];
   const char** local_preload_modules = new const char*[nargs];
+#if HAVE_OPENSSL
+  bool use_bundled_ca = false;
+  bool use_openssl_ca = false;
+#endif  // HAVE_INSPECTOR
```

Comment:
Ah thanks, I'll fix that (https://github.com/nodejs/node/pull/12688). 

And good point, I just assumed that the other PR had been merged, but it does not seem like it. I'll add it to this one and reorder them.